eng/pipeline.yml

```diff
@@ -205,3 +205,34 @@ jobs:
         #
         condition: and(succeeded(), eq(variables['_HelixPipeline'], 'true') ,eq(variables['_BuildConfig'], 'Release'), eq(variables['_PublicBuildPipeline'], 'true'), eq(variables['_ContinuousIntegrationTestsEnabled'], 'true'))
 
+      - task: PowerShell@2
+        displayName: Replace WPF binaries
+        inputs:
+          targetType: 'inline'
+          script: '.\eng\copy-wpf.ps1 -testhost -destination .dotnet -$(_BuildConfig) -arch $(_Platform)'
+          
+      - task: DownloadBuildArtifacts@1
+        displayName: Fetch Test Binaries
+        inputs:
+          buildType: 'specific'
+          project: '9ee6d478-d288-47f7-aacc-f6e6d082ae6d'
+          pipeline: '1145'
+          buildVersionToDownload: 'specific'
+          buildId: '1867477'
```

Comment:
Why lock this in to a single Build ID?  That will result in having to make a PR every time you update the test base

Reply:
As of now the wpf-test repo is in a high velocity test addition mode, we don't want the tests to show as failing. Currently fixing it to a specific build where we know everything is fine. Finally this will be modified to the latest builds.

Reply:
Sure! Thanks @mmitche

Reply:
I hope that after 30 days would be at least one more stable, and if that value breaks it would serve as good reminder to switch to latest builds.